src/components/modal/ModalComposite.tsx

```diff
@@ -112,4 +115,13 @@ export class ModalComposite extends React.PureComponent<IModalCompositeProps & P
     )
 
     private getParent = (): HTMLElement => document.querySelector(Defaults.MODAL_ROOT);
+
+    private handleRequestClose = () => {
+        // Workaround for https://github.com/reactjs/react-modal/issues/745
+        // Once issue is fixed, use onAfterClose={this.props.closeCallback} instead
+        callIfDefined(this.props.onClose);

```

Comment:
do you think Its a good idea to use the onClose to return a boolean if we need or not to do the closeCallback ?

Reply:
I don't really see the added value. If you have a use case where you don't want the `closeCallback` prop to be called afterClose, just don't provide the `closeCallback` prop to the `ModalComposite`.